core/cloudflow-operator/src/main/scala/cloudflow/operator/action/AppActions.scala

```diff
@@ -81,7 +81,17 @@ object AppActions {
         ownerReferences = ownerReferences
       ),
       kind = "Role",
-      rules = List(createEventPolicyRule)
+      rules = List(
+        PolicyRule(
```

Comment:
minor, would be nice to add a private method `createAkkaClusterPolicyRule`, so we can remember what it is for.

Reply:
good call